server/instances.go

```diff
@@ -151,6 +151,28 @@ func (p *Plugin) InstallInstance(instance Instance) error {
 				}
 			}
 
+			if instance.Common().Type == CloudOAuthInstanceType && len(instances.IDs()) > 0 && instances.checkIfExists(instance.GetID()) {
+				p.API.LogDebug("Getting the stored JWT intance data to store inside OAuth instance data for storing its data in the OAuth instance.")
```

Comment:
Please formulate this sentence better its hard to understand just from reading this. 
```suggestion
				p.API.LogDebug("Getting the stored JWT instance data to store inside OAuth instance data for storing its data in the OAuth instance.")
```